frontend/src/components/fundingAgreements/ManageFundingAgreements.vue

```diff
@@ -106,6 +131,55 @@ export default {
     navigateToViewFundingAgreement(id) {
       this.$router.push(`${PATHS.ROOT.VIEW_FUNDING_AGREEMENT}/${id}`);
     },
+    goToChangeRequestHistory() {
+      this.$router.push(PATHS.ROOT.CHANGE_LANDING + '#change-request-history');
+    },
+    getStatusClass(status) {
+      switch (status) {
+        case FUNDING_AGREEMENTS_STATUS.DRAFTED:
+        case FUNDING_AGREEMENTS_STATUS.DRAFTED_PROVIDER_ACTION_REQUIRED:
+        case FUNDING_AGREEMENTS_STATUS.DRAFTED_WITH_MINISTRY:
+          return 'status-gray';
+        case FUNDING_AGREEMENTS_STATUS.REPLACED:
+          return 'status-yellow';
+        case FUNDING_AGREEMENTS_STATUS.ACTIVE:
+          return 'status-blue';
+        case FUNDING_AGREEMENTS_STATUS.APPROVED:
+          return 'status-green';
+        case FUNDING_AGREEMENTS_STATUS.EXPIRED:
+          return 'status-purple';
+        case FUNDING_AGREEMENTS_STATUS.CANCELLED:
+        case FUNDING_AGREEMENTS_STATUS.SUSPENDED:
+        case FUNDING_AGREEMENTS_STATUS.TERMINATED:
+          return 'status-red';
+        default:
+          return '';
+      }
+    },
+    sortFundingAgreements() {
+      this.fundingAgreements?.sort((a, b) => {
+        // 1. Prioritize "Drafted – Provider Action Required" over "Drafted - With Ministry"
+        const statusPriority = {
```

Comment:
This const should be defined outside of the compare function since it really just constant that doesn't depend on the values being sorted.